hand_eye_calibration_target_extractor/src/target_extractor.cc

```diff
@@ -232,8 +232,8 @@ int main(int argc, char** argv) {
       }
     }
 
-    aslam::Camera::ConstPtr camera =
-        aslam::Camera::loadFromYaml(FLAGS_eval_camera_yaml);
+    aslam::Camera::Ptr camera;
+    camera->deserializeFromFile(FLAGS_eval_camera_yaml);
```

Comment:
camera is nullptr.
calling deserializeFromFile from a nullptr causes segmentation fault

cc: @huberya  @HannesSommer @christian-rauch